.github/workflows/publish_npm.yml

```diff
@@ -0,0 +1,29 @@
+name: 'publish npm package'
+
+on: [workflow_dispatch]
+
+jobs:
+  release:
+    name: publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check out the repo'
+        uses: 'actions/checkout@v3'
+        
+      - name: 'Install Node.js'
+        uses: 'actions/setup-node@v3'
+        with:
+          cache: 'yarn'
+          node-version: '16'
```

Comment:
should we use 18 here as it's the current LTS?

Reply:
sure